beacon_node/lighthouse_network/src/service/mod.rs

```diff
@@ -817,17 +817,8 @@ impl<E: EthSpec> Network<E> {
         // unsubscribe from the topic
         let libp2p_topic: Topic = topic.clone().into();
 
-        match self.gossipsub_mut().unsubscribe(&libp2p_topic) {
-            Err(_) => {
-                warn!(self.log, "Failed to unsubscribe from topic"; "topic" => %libp2p_topic);
-                false
-            }
-            Ok(v) => {
-                // Inform the network
-                debug!(self.log, "Unsubscribed to topic"; "topic" => %topic);
-                v
-            }
-        }
```

Comment:
Why did we remove these logs?

Reply:
cause we no longer err on `unsubscribe()` it always succeeds at the `Behavior` level